package.json

```diff
@@ -11,9 +11,18 @@
     "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.2.0",
+    "@fortawesome/free-brands-svg-icons": "^6.2.0",
+    "@fortawesome/free-regular-svg-icons": "^6.2.0",
+    "@fortawesome/free-solid-svg-icons": "^6.2.0",
+    "@fortawesome/vue-fontawesome": "^3.0.2",
+    "axios": "^1.1.3",
+    "mosha-vue-toastify": "^1.0.23",
     "pinia": "^2.0.23",
+    "vee-validate": "^4.7.2",
     "vue": "^3.2.41",
-    "vue-router": "^4.1.5"
+    "vue-router": "^4.1.5",
+    "yup": "^0.32.11"
```

Comment:
what is `yup` for?

Reply:
Yup works with vee-validate to give chainable validation with the composition api

Reply:
Let's us use Formkit for ours forms, so we don't need this.